dids/build.gradle.kts

```diff
@@ -11,33 +11,46 @@ repositories {
   maven("https://repository.jboss.org/nexus/content/repositories/thirdparty-releases/")
 }
 
-val ktor_version = "2.3.4"
-val jackson_version = "2.14.2"
-
 dependencies {
-  api("decentralized-identity:did-common-java:1.9.0")
 
+  /**
+   * Maintainers - please do not declare versioning here at the module level;
+   * versioning is centralized for the platform in $projectRoot/gradle/libs.versions.toml
+   *
+   * Deps are declared in alphabetical order.
+   */
+
+  // API
+  /*
+   * API Leak: https://github.com/TBD54566975/web5-kt/issues/231
+   *
+   * Change and move to "implementation" when completed
+   */
+  api(libs.decentralizedIdentityDidCommonJava)
+
+  // Project
   implementation(project(":common"))
   implementation(project(":crypto"))
-  implementation("com.fasterxml.jackson.module:jackson-module-kotlin:$jackson_version")
-  implementation("com.nimbusds:nimbus-jose-jwt:9.34")
-  implementation("com.github.multiformats:java-multibase:1.1.0")
-  implementation("io.github.oshai:kotlin-logging-jvm:6.0.2")
-
-  implementation("io.ktor:ktor-client-core:$ktor_version")
-  implementation("io.ktor:ktor-client-okhttp:$ktor_version")
-  implementation("io.ktor:ktor-client-content-negotiation:$ktor_version")
-  implementation("io.ktor:ktor-serialization-jackson:$ktor_version")
-
-  implementation("com.squareup.okhttp3:okhttp-dnsoverhttps:4.12.0")
-
-  implementation("io.github.erdtman:java-json-canonicalization:1.1")
 
+  // Implementation
+  implementation(libs.comFasterXmlJacksonModuleKotlin)
+  implementation(libs.comNimbusdsJoseJwt)
+  implementation(libs.comGithubMultiformats)
+  implementation(libs.comSquareupOkhttp3)
+  implementation(libs.dnsJava)
+  implementation(libs.ioGithubErdtmanJavaJsonCanonicalization)
+  implementation(libs.ioGithubOshaiKotlinLogging)
+  implementation(libs.bundles.ioKtorForDids)
+
+  // Test
+  /**
+   * Test dependencies may declare direct versions; they are not exported
+   * and therefore are within the remit of this module to self-define
+   * if desired.
+   */
   testImplementation(kotlin("test"))
-  testImplementation("io.ktor:ktor-client-mock:$ktor_version")
+  testImplementation(libs.ioKtorClientMock)
   testImplementation("org.mockito.kotlin:mockito-kotlin:5.1.0")
```

Comment:
Up to you! For test deps, I'm leaving it to the project devs to determine if you'd like to uplevel into `libs.versions.toml`. I left them like this because as test deps, they're not exported to the runtime or compile `classpath`s of consumers, and these are therefore with the remit of subprojects to either define for themselves or share.